### ISSUE TITLE
Unusable on mobile

### ISSUE THREAD

**Issue by ghost** [processing/p5.js-web-editor#352]
I don't know if it's just my phone, but everytime I press space, the word I enter gets deleted.

For reference, my phone is a Moto E, running android 5.1

**Comment by catarak** [comment]
At the moment the web editor does not support mobile! We hope to support it in the future.

**Comment by slowizzm** [comment]
I would love to see mobile become a higher priority. I use the p5.js for iOS app all the time and would love to just be able to code straight in the web editor where we have access to p5 libraties and able to upload files.

**Comment by catarak** [comment]
i hear that. I've never wanted to do any programming on my phone, so perhaps it is my own bias that has put it lower priority. since the release, a lot of folks have mentioned it. i've never used the iOS app, is it [this one](https://itunes.apple.com/us/app/p5*/id1080192373)?

it would be great to prioritize mobile stuff! I feel like a good first step would be a stripped-down version that works, adding more intricate or complicated details later. honestly, there are a lot of issues that i still think are higher priority that i will be working on first, so i doubt i'll get to this within the next couple months. a great opportunity for someone else to take the reins!

**Comment by slowizzm** [comment]
Yes, that is the correct app. As you can see it is pretty nice, but limited. I used it a bunch in college while I was waiting on classes to start, during down times, ect, as well then and now when I am away from my pc and/or macbook and either have an idea I want to prototype or just want to do some creative coding. 

I have suggested it to my students because it was very helpful for me to practice while I was learning, but the only downfall is that the app doesn’t allow importing of libraries, img, vid, audio, or fonts like the wed editor does.

So for someone like me and/or my students, it would be nice to just use the web editor. 

At the moment it seems the only thing that is making that impossible to do is the responsivness is lacking, the canvas gets cut off in both portrait and landscape. 

I know you have much more pressing issues to deal with, so no worries, just wanted to put it out there. 

I may, if I can find time, pull the source and try to see what I can do to help.

Thanks

**Comment by catarak** [comment]
As a side note, if you want to use assets in your sketches without uploading them, you can use this tool called [CORS anywhere](https://cors-anywhere.herokuapp.com/). You can link to any asset that's already on the Internet, and it will do all of the CORS magic for you. It's a little slow but could be great on mobile. There's a world in which we could add a CORS anywhere server to the current deployment, and maybe that would be way faster.

**Comment by catarak** [comment]
Even though the mobile version is still a work in progress and not deployed to production, I think this is fixed in the current mobile version.
